pkg/spoke/managedcluster/creating_controller.go

```diff
@@ -68,6 +72,9 @@ func (c *managedClusterCreatingController) sync(ctx context.Context, syncCtx fac
 		ObjectMeta: metav1.ObjectMeta{
 			Name: c.clusterName,
 		},
+		Spec: clusterv1.ManagedClusterSpec{
+			LeaseDurationSeconds: int32(LeaseDurationSeconds),
```

Comment:
We have a [mutating webhook](https://github.com/open-cluster-management-io/registration/blob/main/pkg/webhook/cluster/mutating_webhook.go#L62) to set this value, and in your case, I guess you may not start the webhook, right? so I think we may not do this, what's your opinion?

Reply:
Yeal, i don't install webhook compoment. If set default values will not use this, i will remove this logic soon.